### ISSUE TITLE
Is this project still actively maintained?

### ISSUE THREAD

**Issue by bhipple** [hoxu/gitstats#67]
It looks like there are a large number of PRs unanswered.  Is there a fork we should be using instead?

**Comment by PanderMusubi** [comment]
Just added a fix for this fork https://github.com/laserb/gitstats/pull/2

**Comment by sebix** [comment]
Does anyone have a fork with all fixes applied? A lot of work has been done by many contributors which is waiting to be integrated.

**Comment by PanderMusubi** [comment]
Nope. I guess one has to be selected with confirmation of the author of that PR that he or she would like to main this.

**Comment by pulkomandy** [comment]
See https://github.com/vifactor/repostat for an updated version.

**Comment by julianmcolina** [comment]
Not sure if there is a working fork for this project (it kind of seems stale), but [Haystack](https://usehaystack.io/?utm_source=github&utm_medium=gitstats&utm_campaign=is%20this%20project%20still%20maintained) has a service version that may be helpful

p.s. I am a contributor to that company
